interp/interp.go

```diff
@@ -335,9 +337,27 @@ func (interp *Interpreter) Repl(in, out *os.File) {
 	}
 }
 
+// trapEval is a wrapper around Eval that traps a panic if the input is a terminal
+func (interp *Interpreter) trapEval(inTerminal bool, src string) (reflect.Value, error) {
```

Comment:
@mvertes I'm not sure about the naming `trapEval` (seems too "bash" for me) WDYT?

Reply:
Yeah, on the lint part...

I did not make any change to yaegi.go
And in get prompt it was already "out" for the second parameter.

As far as "trapEval"...

recoveredEval?
checkedEval?
userEval?

```
$ golangci-lint run
cmd/yaegi/yaegi.go:1: : 
cmd/yaegi/test.go:1:1: expected 'package', found 1 (typecheck)
/*
cmd/yaegi/yaegi.go:32:2: could not import flag (no metadata for flag) (typecheck)
	"flag"
	^
cmd/yaegi/yaegi.go:33:2: could not import fmt (no metadata for fmt) (typecheck)
	"fmt"
	^
interp/interp.go:359:33: `out` can be `io.Writer` (interfacer)
func getPrompt(inTerminal bool, out *os.File) func() {
```

Reply:
After reviewing Entering 1=1 triggers a panic #280 some more I don't really believe this PR will really truly fix the underlying problem. It does make it more manageable, but one underlying issue is the panics are not recovered for correctly. Something else gets messed up. See my further comments on #280

Reply:
yes your PR don't fix #280, as you say: make the panic more manageable.